src/test/java/uk/gov/companieshouse/web/accounts/token/impl/TokenManagerImplTests.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.companieshouse.web.accounts.token.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.security.SignatureException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.environment.EnvironmentReader;
+import uk.gov.companieshouse.web.accounts.token.TokenManager;
+
+@ExtendWith(MockitoExtension.class)
+public class TokenManagerImplTests {
+
+    private static final String CHS_JWT_SECRET_ENV_VAR = "CHS_JWT_SECRET";
+
+    private static final String CHS_JWT_SECRET = "ZTljOTQ3MTdkN2Y0YmUwOTJjZDA0OTJmMTQ4OTMwODM=";
+
+    @Mock
+    private EnvironmentReader environmentReader;
```

Comment:
For additional clarity it would be good if Mock objects were prefixed/suffixed with 'mock'.